packages/react-components/react-tag-picker-preview/stories/TagPicker/TagPickerBigText.stories.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
+import {
+  TagPicker,
+  TagPickerList,
+  TagPickerInput,
+  TagPickerControl,
+  TagPickerProps,
+  TagPickerOption,
+  TagPickerGroup,
+} from '@fluentui/react-tag-picker-preview';
+import { Tag, Avatar, makeStyles } from '@fluentui/react-components';
+
+const options = [
+  'John Doe',
+  'Jane Doe',
+  'Max Mustermann',
+  'Erika Mustermann',
+  'Pierre Dupont',
+  'Amelie Dupont',
+  'Lorem ipsum dolor sit amet consectetur adipisicing elit. Possimus ut corrupti totam odit nulla ipsa perferendis quasi, soluta mollitia autem esse quod molestiae. Alias, atque libero debitis iusto delectus aliquam!',
+  'Maria Rossi',
+];
+
+const useStyles = makeStyles({
+  tagPrimaryText: {
+    whiteSpace: 'nowrap',
+    overflowX: 'hidden',
+    textOverflow: 'ellipsis',
+  },
+  optionContent: {
+    whiteSpace: 'nowrap',
+    overflowX: 'hidden',
+    textOverflow: 'ellipsis',
+  },
+  optionSecondaryContent: {
+    whiteSpace: 'nowrap',
+  },
+});
+
+export const BigText = () => {
```

Comment:
Let's add some documentation here that this is **one way** of handling ellipsis and that it basically makes one tag take up the entire width of the control

Reply:
Sounds good to me. I'll do both